src/icon/Icons.tsx

```diff
@@ -961,6 +1040,107 @@ export const ExpandOutline = () => (
   </svg>
 );
 
+export const Explainability = () => (
+  <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+  >
+      <rect
+          x="10.75"
+          y="12"
+          width="2.5"
+          height="9.98733"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="10.75"
+          y="2.00096"
+          width="2.5"
+          height="8.27023"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="14.5417"
+          y="8.77119"
+          width="2.5"
+          height="11.027"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="18.3337"
+          y="11.5279"
+          width="2.5"
+          height="6.93593"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="18.3337"
+          y="5.89282"
+          width="2.5"
+          height="3.68917"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="2.98645"
+          y="14.8522"
+          width="2.5"
+          height="3.61164"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="2.98645"
+          y="5.89283"
+          width="2.5"
+          height="7.3378"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="6.77844"
+          y="2.97392"
+          width="2.5"
+          height="9.52698"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="6.77844"
+          y="14.3252"
+          width="2.5"
+          height="6.44591"
+          rx="1.25"
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="14.5417"
+          y="3.37933"
+          width="2.5"
+          height="3.64864"
+          rx="1.25"
+          fill="white"
```

Comment:
We need to delete all these "white" hard codings. If you run the the storybook the colors will probably not rotate.

Reply:
took a minute but removed all the fills! had to regenerate some of these icons since figma automatically includes it on export